plugins/node/opentelemetry-instrumentation-aws-sdk/package.json

```diff
@@ -48,29 +48,32 @@
   "dependencies": {
     "@opentelemetry/core": "^1.0.0",
     "@opentelemetry/instrumentation": "^0.27.0",
+    "@opentelemetry/propagation-utils": "^0.26.0",
     "@opentelemetry/semantic-conventions": "^1.0.0",
-    "@opentelemetry/propagation-utils": "^0.26.0"
+    "aws-lambda": "^1.0.7"
```

Comment:
is it used anywhere?

Reply:
It was used for the ClientContext type, however since this was removed I've now removed the package.